src/main/java/com/softserveinc/ita/pageobjects/admin/ProtocolPage.java

```diff
@@ -1,4 +1,48 @@
 package com.softserveinc.ita.pageobjects.admin;
 
+import com.codeborne.selenide.SelenideElement;
+
+import static com.codeborne.selenide.Selenide.$x;
+import static java.lang.String.format;
+
 public class ProtocolPage extends MainMenu {
+
+    private final SelenideElement startDatePickerButton = $x("(//button[@aria-label='Open calendar'])[1]");
```

Comment:
could be as DATE_PICKER_TEMPLATE = "(//button[@aria-label='Open calendar'])[%s]";